### PR TITLE
Remove console.log

### PR DESCRIPTION
**Pull request by kiwiupover**

### CODE REVIEW

File: tests/unit/index-nodetest.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ chai.use(chaiAsPromised);
 
 var assert = chai.assert;
 
-describe('s3 plugin', function() {
+describe('deploy-asset-sizes plugin', function() {
```
Comment (YoranBrondsema) [inline tests/unit/index-nodetest.js:9]: Good one :-)

**Comment by kiwiupover** [comment]
@YoranBrondsema thanks for the tests travis is now setup

**Comment by YoranBrondsema** [comment]
@kiwiupover Perfect, thanks for setting that up!

**Comment by kiwiupover** [comment]
FYI there are two npm script to deploy the app with tags the release. 

https://github.com/kiwiupover/ember-cli-deploy-asset-sizes/blob/master/package.json#L13-L14

**Comment by YoranBrondsema** [comment]
:+1:

**Pull request merged**